spec/support/standard_setup.rb

```diff
@@ -0,0 +1,161 @@
+class StandardSetup
+  include FactoryGirl::Syntax::Methods
+
+  attr_reader :cases, :user_teams
+  # rubocop:disable Metrics/MethodLength
+
+  def initialize
+    @teams = {
+       disclosure_bmt_team:       find_or_create(:team_dacu),
+       disclosure_team:           find_or_create(:team_dacu_disclosure),
+       responding_team:           create(:responding_team),
+       press_office_team:         find_or_create(:team_press_office),
+       private_office_team:       find_or_create(:team_private_office),
+       another_approving_team:    create(:approving_team)
+    }
+
+
+    @users = {
+      manager_user:                         find_or_create(:disclosure_bmt_user),
+      approver_user:                        find_or_create(:disclosure_specialist),
+      another_approver_user:                create(:approver, approving_team: another_approving_team),
```

Comment:
Shouldn't we be testing for approver coworkers too? Similar to the `another_responder_in_same_team_user` below (which might also be expressed as `responder_coworker_user`).

Reply:
done